JMS/com/ibm/mq/samples/jms/JwtHelper.java

```diff
@@ -0,0 +1,93 @@
+/*
+* (c) Copyright IBM Corporation 2024
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+package com.ibm.mq.samples.jms;
+
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpRequest.BodyPublishers;
+import java.util.logging.Logger;
+import java.net.http.HttpResponse;
+import org.json.JSONObject;
+import com.ibm.mq.samples.jms.SampleEnvSetter;
+
+public class JwtHelper {
+    private static final Logger logger = Logger.getLogger("com.ibm.mq.samples.jms");
+    private static String tokenEndpoint = "";
+    private static String tokenUsername = "";
+    private static String tokenPassword = "";
+    private static String tokenClientId = "";
+    
+    public JwtHelper(SampleEnvSetter env) {
+        if (tokenEndpoint.trim().isEmpty()) {
```

Comment:
I was thinking of letting this go, but it sticks out too much. The normal way to flag whether static values have been set, is to see a static initialised boolean, which defaults to false. Then there is only one if block.